clientconn.go

```diff
@@ -525,8 +526,8 @@ func chainStreamClientInterceptors(cc *ClientConn) {
 		chainedInt = func(ctx context.Context, desc *StreamDesc, cc *ClientConn, method string, streamer Streamer, opts ...CallOption) (ClientStream, error) {
 			return interceptors[0](ctx, desc, cc, method, getChainStreamer(interceptors, 0, streamer), opts...)
 		}
+		cc.dopts.streamInt = chainedInt
```

Comment:
This seems like an unrelated change. Why is this required?

Reply:
I'm sorry, this is an accident.
I've reverted this change.